orchestration/templates/extract-from-json.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: extract-from-json
+spec:
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: pvc-name
+          - name: file-path
+          - name: json-key
+      volumes:
+        - name: state
+          persistentVolumeClaim:
+            claimName: '{{ "{{inputs.parameters.pvc-name}}" }}'
+            readOnly: true
+      {{- include "argo.retry" . | indent 6 }}
+      script:
+        image: python3-slim
+        command: [python]
+        source: |
+          python3 "scripts/extract-json-value.py" \
+          - --file "{{inputs.parameters.file-path}}" \
```

Comment:
A few things to change here:
1. The leading `-`s aren't needed, because this is a multi-line string
2. Since we're executing `python` as the command, the value for `source` must be valid python code. The bash-isms like trailing `\`s aren't needed.